frontend/src/utils/fileArrayBuffer.ts

```diff
@@ -0,0 +1,29 @@
+// Backward-compatible version of of array buffer function.
+
+function promisifyFile(obj: FileReader): Promise<ArrayBuffer> {
+  return new Promise(function (resolve, reject) {
+    obj.onload = obj.onerror = function (evt) {
+      obj.onload = obj.onerror = null
+      console.log(obj.result)
+      evt.type === 'load' && obj.result
+        ? resolve(obj.result as ArrayBuffer)
+        : reject(new Error('Failed to read the blob/file'))
+    }
+  })
+}
+
+const fileArrayBuffer = async (file: File): Promise<ArrayBuffer> => {
+  let buffer
+  // arrayBuffer is only compatible with Safari 14 onwards
+  // for older browsers, use readAsArrayBuffer
+  if (!file.arrayBuffer) {
+    const fr = new FileReader()
+    fr.readAsArrayBuffer(file)
+    buffer = await promisifyFile(fr)
+  } else {
+    buffer = await file.arrayBuffer()
+  }
```

Comment:
/superNit to save on negative thinking and saving 1 byte of delivery 😅 
```suggestion
  if (file.arrayBuffer) {
    buffer = await file.arrayBuffer()
  } else {
    const fr = new FileReader()
    fr.readAsArrayBuffer(file)
    buffer = await promisifyFile(fr)
  }
```

Reply:
[0d85540](https://github.com/opengovsg/FormSG/pull/6042/commits/0d8554063b4fde8441e9a196d7780edd9d5b0d85)